docs/db/changelog/changesets/powerauth-java-server/1.6.x/20231018-add-constraint-operation-template-name.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.9.xsd">
+
+    <changeSet id="1" logicalFilePath="powerauth-java-server/1.6.x/20231018-add-constraint-operation-template-name.xml" author="Jan Pesek">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <uniqueConstraintExists tableName="pa_operation_template" columnNames="template_name" constraintName="pa_operation_template_template_name_uk" />
+            </not>
+        </preConditions>
+        <comment>Add unique constraint to pa_operation_template.template_name</comment>
+        <addUniqueConstraint tableName="pa_operation_template" columnNames="template_name" constraintName="pa_operation_template_template_name_uk" />
```

Comment:
@zcgandcomp Should we keep recording the db changes also in the migration guides or not?

Reply:
We should add information into the migration guide, please add this information in the usual way as seen in previous migration guides.

Reply:
Should we be concrete about the changes or just mention that some changes are needed and link a script, which will be generated during the release?

Reply:
https://github.com/wultra/development-internal/wiki/Using-Liquibase#13-migration-between-versions. Yes please add a migration guide and mention the change. In theory it is possible to have (invalid) duplicities in the DB. The migration script in SQL will be generated during release procedure.